docs/rest_api.md

```diff
@@ -479,30 +484,42 @@ Authorization requirements:
 
 `POST /node/<node>/power_cycle`
 
+Request Body:
+
+    {
+        "force": <boolean> (Optional, defaults to False)
+    }
+
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
 Accepts one optional boolean argument that determines whether to soft (default)
 or hard reboot the system.
 
-Request body:
-    {
-    	"force": <boolean> (Optional, defaults to False)
-    }
+Authorization requirements:
+
+* If the owner is a project, access to that project is required.
```

Comment:
I'm assuming this wording was copied from one of the networking related calls. "If the owner is a project" is a bit awkward in this case, since nodes can only be owned by projects (or be "free"), whereas with networks we have a notion of an admin-owned network. Maybe something more like:

* If the node is assigned to a project, access to that project is required.

Same goes for the other call you added this to.

Reply:
Changed this